static/app/views/eventsV2/table/quickContext.tsx

```diff
@@ -287,11 +303,47 @@ function ReleaseContext(props: ReleaseContextProps) {
   }
 
   return (
-    <Fragment>
+    <Wrapper data-test-id="quick-context-hover-body">
       {renderReleaseDetails()}
       {renderIssueCountAndAuthors()}
       {renderLastCommit()}
-    </Fragment>
+    </Wrapper>
+  );
+}
+
+function EventContext(props: BaseContextProps) {
+  let stackTrace: StacktraceType | null = null;
```

Comment:
nit: this doesn't have to be declared up here, better to declare as a `const` where it's being used below for readability

Reply:
Agree